internal/sql/repository/AppListingRepository.go

```diff
@@ -138,6 +145,21 @@ func (impl AppListingRepositoryImpl) FetchOverviewAppsByEnvironment(envId, limit
 	return envContainers, err
 }
 
+func (impl AppListingRepositoryImpl) FetchLastDeployedImage(appId, envId int) (*LastDeployed, error) {
```

Comment:
this function returning the last deployed image, this must be somewhere with similar output.